.github/workflows/deploy.yml

```diff
@@ -0,0 +1,241 @@
+name: Deploy
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  deploy_protectli:
+    runs-on: ubuntu-22.04
+    if: contains(github.ref, 'refs/tags/protectli')
+    strategy:
+      matrix:
+        vendor: [ protectli ]
+        model: [ vp66xx, vp46xx, vp2420, vp2410, V1210, V1211, V1410, V1610 ]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Checkout all submodules
+        run: git submodule update --init --recursive --checkout
+
+      - name: Set up tag name
+        id: tag_name
+        run: echo "tag=${GITHUB_REF#refs/tags/}" >> "$GITHUB_OUTPUT"
+
+      - name: Parse directories from tag
+        id: parse_directories
+        run: |
+          tag=${{ steps.tag_name.outputs.tag }}
+          base_dir="protectli"
+          model=$(echo "$tag" | cut -d'_' -f1-3)
+          release=$(echo "$tag" | cut -d'_' -f4)
+          echo "base_dir=$base_dir" >> "$GITHUB_OUTPUT"
+          echo "model=$model" >> "$GITHUB_OUTPUT"
+          echo "release=$release" >> "$GITHUB_OUTPUT"
+
+      # Allows downloading artifacts from a different workflow
+      - name: Download workflow artifact
+        uses: dawidd6/action-download-artifact@v6
```

Comment:
Why do we need to use action from some user, not from GH: https://github.com/actions/download-artifact ?

Reply:
The "official" action does not allow downloading artifacts from a different workflow.